pkg/aws/awscluster/mutate_awscluster.go

```diff
@@ -102,12 +111,25 @@ func (m *Mutator) MutateUpdate(request *admissionv1.AdmissionRequest) ([]mutator
 	if _, _, err = mutator.Deserializer.Decode(request.OldObject.Raw, nil, awsClusterOld); err != nil {
 		return nil, microerror.Maskf(parsingFailedError, "unable to parse old AWSCluster: %v", err)
 	}
+
 	patch, err = m.MutatePodCIDR(*awsCluster)
 	if err != nil {
 		return nil, microerror.Mask(err)
 	}
 	result = append(result, patch...)
 
+	patch, err = m.MutateDescription(*awsCluster)
```

Comment:
why adding MutateDescription when it's about DnsDOmain

Reply:
I realized that I forgot to add it to the update mutator in my other PR. :)